Modules/Devices/src/Devices.Application/Identities/IdentityDeleter.cs

```diff
@@ -14,9 +14,11 @@ public IdentityDeleter(IMediator mediator)
         _mediator = mediator;
     }
 
-    public async Task Delete(IdentityAddress identityAddress)
+    public async Task Delete(IdentityAddress identityAddress, IDeletionProcessLogger deletionProcessLogger)
     {
         await _mediator.Send(new DeletePnsRegistrationsOfIdentityCommand(identityAddress));
+        await deletionProcessLogger.LogDeletion(identityAddress, AggregateType.PnsRegistrations);
```

Comment:
Be consistent: log after or before command in each case.

Reply:
Changes made.